# Point cloud builder
accumulator_number_persistant_clouds : 10

# Filter
persistant_cloud_filter_min_neighbors : 10
persistant_cloud_filter_radius : 0.1

# Clusterer
cluster_tolerance_m : 4.4
cluster_min_points : 20
cluster_max_points : 1000

# Associator
associator_max_distance : 5

# Ogrid
ogrid_height_meters : 300
ogrid_width_meters : 300
ogrid_resolution_meters_per_cell : 0.3
ogrid_inflation_meters : 0.5
